bcipy/signal/model/offline_analysis.py

```diff
@@ -137,11 +138,12 @@ def offline_analysis(
     # Channel map can be checked from raw_data.csv file or the devices.json located in the acquisition module
     # The timestamp column [0] is already excluded.
     channel_map = analysis_channels(channels, type_amp)
+    data, fs = raw_data.by_channel()
 
     inquiries, inquiry_labels, inquiry_timing = model.reshaper(
         trial_targetness_label=trigger_targetness,
         timing_info=trigger_timing,
-        eeg_data=raw_data.by_channel(),
+        eeg_data=data,
         fs=sample_rate,
```

Comment:
We did this to avoid re-assigning to the same name after transformation: https://github.com/CAMBI-tech/BciPy/blob/86692d099f1c5857de858fe42ce2637280bb3bdd/bcipy/signal/model/offline_analysis.py#L155

There are cases we might need the original and also transformed sample rates. I think sample_rate may be a better kwarg name as it's explicit. What do you think?